[
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11259",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11259/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11259/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11259/events",
    "html_url": "https://github.com/npm/npm/issues/11259",
    "id": 128326963,
    "number": 11259,
    "title": "getting error",
    "user": {
      "login": "h4harish",
      "id": 14356987,
      "avatar_url": "https://avatars.githubusercontent.com/u/14356987?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/h4harish",
      "html_url": "https://github.com/h4harish",
      "followers_url": "https://api.github.com/users/h4harish/followers",
      "following_url": "https://api.github.com/users/h4harish/following{/other_user}",
      "gists_url": "https://api.github.com/users/h4harish/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/h4harish/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/h4harish/subscriptions",
      "organizations_url": "https://api.github.com/users/h4harish/orgs",
      "repos_url": "https://api.github.com/users/h4harish/repos",
      "events_url": "https://api.github.com/users/h4harish/events{/privacy}",
      "received_events_url": "https://api.github.com/users/h4harish/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 1,
    "created_at": "2016-01-23T12:52:18Z",
    "updated_at": "2016-01-23T15:03:03Z",
    "closed_at": null,
    "body": "My command is \r\nnpm install \r\nWhen am gonna to run this command in on my webstorm terminol it going well but after am typing \r\nnpm run build \r\nIt's going to throw me error \r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!     C:\\Users\\harish\\WebstormProjects\\Angular 2 first Attemt\\npm-debug.log\r\n\r\nHow to solve this Error,please give a Solution\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11258",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11258/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11258/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11258/events",
    "html_url": "https://github.com/npm/npm/issues/11258",
    "id": 128324155,
    "number": 11258,
    "title": "Rename username",
    "user": {
      "login": "d10",
      "id": 813865,
      "avatar_url": "https://avatars.githubusercontent.com/u/813865?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/d10",
      "html_url": "https://github.com/d10",
      "followers_url": "https://api.github.com/users/d10/followers",
      "following_url": "https://api.github.com/users/d10/following{/other_user}",
      "gists_url": "https://api.github.com/users/d10/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/d10/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/d10/subscriptions",
      "organizations_url": "https://api.github.com/users/d10/orgs",
      "repos_url": "https://api.github.com/users/d10/repos",
      "events_url": "https://api.github.com/users/d10/events{/privacy}",
      "received_events_url": "https://api.github.com/users/d10/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-01-23T11:57:33Z",
    "updated_at": "2016-01-23T11:57:33Z",
    "closed_at": null,
    "body": "Is there a way to rename username? I couldn't find it in the npm site/issues."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11257",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11257/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11257/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11257/events",
    "html_url": "https://github.com/npm/npm/issues/11257",
    "id": 128321107,
    "number": 11257,
    "title": "problem installing ionic on windows?",
    "user": {
      "login": "ved54321",
      "id": 7689116,
      "avatar_url": "https://avatars.githubusercontent.com/u/7689116?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ved54321",
      "html_url": "https://github.com/ved54321",
      "followers_url": "https://api.github.com/users/ved54321/followers",
      "following_url": "https://api.github.com/users/ved54321/following{/other_user}",
      "gists_url": "https://api.github.com/users/ved54321/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ved54321/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ved54321/subscriptions",
      "organizations_url": "https://api.github.com/users/ved54321/orgs",
      "repos_url": "https://api.github.com/users/ved54321/repos",
      "events_url": "https://api.github.com/users/ved54321/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ved54321/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-01-23T11:02:07Z",
    "updated_at": "2016-01-23T11:02:39Z",
    "closed_at": null,
    "body": "i have install node version 5.4.0 and npm version 3.3.12.While running npm install -g cordova ionic i am getting the following issue,\r\n\r\nC:\\Windows\\system32>npm install -g cordova ionic\r\n'CALL \"C:\\Program Files\\nodejs\\\\node.exe\" \"C:\\Program Files\\nodejs\\\\node_modules\\npm\\b\r\nin\\npm-cli.js\" prefix -g' is not recognized as an internal or external command,\r\noperable program or batch file.\r\nnpm ERR! Windows_NT 6.1.7601\r\nnpm ERR! argv \"C:\\\\Program Files\\\\nodejs\\\\node.exe\" \"C:\\\\Program Files\\\\nodejs\\\\node_m\r\nodules\\\\npm\\\\bin\\\\npm-cli.js\" \"install\" \"-g\" \"cordova\" \"ionic\"\r\nnpm ERR! node v5.4.0\r\nnpm ERR! npm  v3.3.12\r\n\r\nnpm ERR! shasum check failed for C:\\Users\\admin\\AppData\\Local\\Temp\\npm-5768-1aee47db\\r\r\negistry.npmjs.org\\ionic\\-\\ionic-1.7.13.tgz\r\nnpm ERR! Expected: 0a16c0aea3280acd406938d31165fc0f2d258e06\r\nnpm ERR! Actual:   99b3c58da5b825305001e56ce41506fdc1be1c20\r\nnpm ERR! From:     https://registry.npmjs.org/ionic/-/ionic-1.7.13.tgz\r\nnpm ERR!\r\nnpm ERR! If you need help, you may report this error at:\r\nnpm ERR!     <https://github.com/npm/npm/issues>\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!     C:\\Windows\\system32\\npm-debug.log\r\n\r\n\r\nAny idea why i am getting this error ?"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11256",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11256/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11256/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11256/events",
    "html_url": "https://github.com/npm/npm/issues/11256",
    "id": 128280626,
    "number": 11256,
    "title": "npm install sub-dependencies problem.",
    "user": {
      "login": "dfcreative",
      "id": 300067,
      "avatar_url": "https://avatars.githubusercontent.com/u/300067?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/dfcreative",
      "html_url": "https://github.com/dfcreative",
      "followers_url": "https://api.github.com/users/dfcreative/followers",
      "following_url": "https://api.github.com/users/dfcreative/following{/other_user}",
      "gists_url": "https://api.github.com/users/dfcreative/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/dfcreative/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/dfcreative/subscriptions",
      "organizations_url": "https://api.github.com/users/dfcreative/orgs",
      "repos_url": "https://api.github.com/users/dfcreative/repos",
      "events_url": "https://api.github.com/users/dfcreative/events{/privacy}",
      "received_events_url": "https://api.github.com/users/dfcreative/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-01-23T01:09:46Z",
    "updated_at": "2016-01-23T01:14:32Z",
    "closed_at": null,
    "body": "Hello!\r\n\r\nWith the npm@3+ I started getting troubles with `npm install <module>` and `npm update <module>`.\r\n\r\nSuppose a dep structure like that:\r\n```\r\na@1.0.0                  ← we’re working on this guy\r\n├── b@1.0.0\r\n└── c@1.0.0\r\n    └── b@1.0.0\r\n```\r\n\r\nFirst `npm install` is correct.\r\nLater though, if I update `c@1.0.0` to `c@1.0.1`, and do `npm update c` or `npm install c`, I just loose `b` module from my `a` package (`b` migrates to `c` i guess), so I start getting an error `Cannot find module 'b'` and I’m forced to do `npm install` of everything once again. Quite annoying.\r\n\r\nIs there something I misunderstand or an issue I should’ve found and read about that?\r\n\r\nWin7, npm@3.3.12, node@5.3.0\r\n\r\nThank you."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11253",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11253/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11253/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11253/events",
    "html_url": "https://github.com/npm/npm/issues/11253",
    "id": 128213379,
    "number": 11253,
    "title": "$npm_package_version env var is wrong during `preversion` and `version` lifecycle scripts",
    "user": {
      "login": "jasonkarns",
      "id": 119972,
      "avatar_url": "https://avatars.githubusercontent.com/u/119972?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/jasonkarns",
      "html_url": "https://github.com/jasonkarns",
      "followers_url": "https://api.github.com/users/jasonkarns/followers",
      "following_url": "https://api.github.com/users/jasonkarns/following{/other_user}",
      "gists_url": "https://api.github.com/users/jasonkarns/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/jasonkarns/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/jasonkarns/subscriptions",
      "organizations_url": "https://api.github.com/users/jasonkarns/orgs",
      "repos_url": "https://api.github.com/users/jasonkarns/repos",
      "events_url": "https://api.github.com/users/jasonkarns/events{/privacy}",
      "received_events_url": "https://api.github.com/users/jasonkarns/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 1,
    "created_at": "2016-01-22T18:21:49Z",
    "updated_at": "2016-01-22T18:26:20Z",
    "closed_at": null,
    "body": "Given `version` lifecycle scripts:\r\n\r\n``` json\r\n  \"version\": \"1.0.0\",\r\n  \"scripts\": {\r\n    \"preversion\": \"echo $npm_package_version\",\r\n    \"version\": \"echo $npm_package_version\",\r\n    \"postversion\": \"echo $npm_package_version\"\r\n  },\r\n```\r\n\r\nWhen `$ npm version patch`\r\n\r\nThen I should see:\r\n``` bash\r\n> preversion\r\n1.0.0\r\n> version\r\n1.0.0\r\n> postversion\r\n1.0.1\r\n```\r\n\r\nInstead, I see:\r\n``` bash\r\n> preversion\r\n1.0.1\r\n> version\r\n1.0.1\r\n> postversion\r\n1.0.1\r\n```\r\n\r\nIt seems that the `npm_package_version` env var is updated to the new version before the lifecycle scripts are run. The var shouldn't be updated until the `postversion` script (since both `preversion` and `version` scripts are executed BEFORE the version bump).\r\n\r\n\r\nTested with: npm 3.5.3 (latest) and 3.6.0 (next)"
  }
]
